server/jetstream_cluster.go

```diff
@@ -8383,7 +8388,7 @@ RETRY:
 	}
 
 	numRetries++
-	if numRetries >= maxRetries {
```

Comment:
This was an off-by-one. `maxRetries=3`, but there would only be max 2 retries.

Reply:
Story of my life ;)